### PR TITLE
update tool script to use BiocManager instead of online R script

### PR DESCRIPTION
**Pull request by LiNk-NY**
Hi Kirill, @krlmlr 
As you may be aware, Bioconductor now uses `BiocManager` to install and update Bioconductor packages. In the near future, the online `biocLite.R` script may no longer be supported.

_Note_. You may want to use an alternative version of downloading the `BiocManager`,
for example, as part of the CRAN package list. See `BiocManager` on CRAN [here](https://CRAN.r-project.org/package=BiocManager).

The current pull request installs `BiocManager` when the `R_USE_BIOC_CMD` is invoked. 

Best regards,
Marcel

### CODE REVIEW

**Comment by jimhester** [comment]
@LiNk-NY If `biocLite.R` is no longer going to be supported what is the maintenance / migration plan for older versions of R? BiocManager requires R 3.5+

**Comment by LiNk-NY** [comment]
Hi all,
I chatted with Martin about this and was told that we will keep `biocLite.R` around for
legacy reasons. 
But we prefer that our users, with compatible versions of R, move over to `BiocManager`.
cc: @mtmorgan

**Comment by LiNk-NY** [comment]
Hi Kirill and Jim, @krlmlr @jimhester 
Please see the updated PR. 
It should be compatible with R versions above and below `3.5.0`.

**Comment by LiNk-NY** [comment]
Any word on merging this? @jimhester @krlmlr

File: r-travis/scripts/travis-tool.sh [hunk 1]
```diff
@@ -27,10 +27,23 @@ PATH="${PATH}:/usr/texbin"
 R_BUILD_ARGS=${R_BUILD_ARGS-"--no-manual"}
 R_CHECK_ARGS=${R_CHECK_ARGS-"--no-manual --as-cran"}
 
-R_USE_BIOC_CMDS="source('${BIOC}');"\
+R_VERSION_TEST="paste(R.Version()$major, R.Version()$minor, sep = '.') >="\
+" package_version('3.5.0')"
```
Comment (krlmlr) [inline r-travis/scripts/travis-tool.sh]: Maybe simpler:

``` r
getRversion() >= "3.5.0"
#> [1] TRUE
```

<sup>Created on 2018-11-28 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1.9000)</sup>

Not sure if this works for older R versions, though.

Comment (LiNk-NY) [reply r-travis/scripts/travis-tool.sh]: I had a look at the history for this function and it's 10 years old so I guess it does support older R versions up to 10 years back.

**Comment by LiNk-NY** [comment]
https://ci.appveyor.com/project/krlmlr/r-appveyor/builds/20659718

**Comment by mbstadler** [comment]
Are there any news regarding the merging of this pull request? It would be great to be able to use appveyor with R >= 3.5.0. @jimhester @krlmlr

**Comment by csoneson** [comment]
Hi @krlmlr, @jimhester - any updates on the merging of this PR? Or is there any other way of using r-appveyor with the current Bioconductor version? Thanks!

**Comment by mjsteinbaugh** [comment]
+1 on updating this -- I'd love to be able to test some of my single-cell RNA-seq code against Bioconductor 3.8 on Windows

**Comment by LiNk-NY** [comment]
Hi Jim and Kirill, @jimhester @krlmlr 
I'd be happy to make any changes to the PR here. 
Let me know. 
Best regards,
Marcel

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**